config/default_settings/zh_CN.yml

```diff
@@ -0,0 +1,11 @@
+posting:
+  min_post_length: 8
+  min_first_post_length: 8
+  min_private_message_post_length: 4
+  min_topic_title_length: 5
+  uncategorized_description: 不需要分类的主题，或者不适用于任何分类的主题。
```

Comment:
Why is this even in the site_settings? Can't we move that into `server.yml` and allow proper translation in Transifex? Wait, there is already a (different) [uncategorized_description in `server.yml`](https://github.com/discourse/discourse/blob/master/config/site_settings.yml#L352).